src/Command/Branch/BranchSyncCommand.php

```diff
@@ -28,18 +30,69 @@ protected function configure()
     {
         $this
             ->setName('branch:sync')
-            ->setDescription('Syncs local branch with its upstream version')
-            ->addArgument('branch_name', InputArgument::OPTIONAL, 'Branch name to sync')
+            ->setDescription('Synchronises a local branch with a remote branch')
+            ->addArgument('source_branch', InputArgument::OPTIONAL, 'Remote branch-name to use as (new) base')
+            ->addArgument('source_remote', InputArgument::OPTIONAL, 'Remote to pull from (defaults to [your-username])')
             ->addArgument(
-                'remote',
+                'dest_remote',
```

Comment:
This is a BC break. Can't you rather add both `remote` and `dest_remote` with a deprecation warning when using `remote`?